tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/DynamicConfigurationTests.cs

```diff
@@ -0,0 +1,256 @@
+// <copyright file="DynamicConfigurationTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.RemoteConfigurationManagement;
+using Datadog.Trace.Telemetry;
+using Datadog.Trace.TestHelpers;
+using Datadog.Trace.Vendors.Newtonsoft.Json;
+using Datadog.Trace.Vendors.Newtonsoft.Json.Linq;
+using FluentAssertions;
+using FluentAssertions.Execution;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests
+{
+    [CollectionDefinition(nameof(DynamicConfigurationTests), DisableParallelization = true)]
+    [Collection(nameof(DynamicConfigurationTests))]
```

Comment:
why a collection? Isn't there only one class with that collection?

Reply:
🤔  that's a mistake I think. I wanted to disable parallelization for the unit test, but I don't think that's needed for the integration test. Removing it.

Reply:
Never mind I remember now, I need to disable parallelization because I use the `LogEntryWatcher` (and all samples log to the same file)